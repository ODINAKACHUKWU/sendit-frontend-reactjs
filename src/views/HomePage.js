import React from "react";
import { HomePageNavBar } from "../components/NavBar";
import Footer from "../components/Footer";
import Button from "../components/common/Button";

import "../styles/Homepage.css";

const HomePage = () => {
  return (
    <div className="row content">
      <HomePageNavBar />
      <div className="main content">
        <div className="home-container">
          <div className="col-5 left">
            <h1>Courier Services</h1>
            <p>
              SendIT is a courier service that helps users deliver parcels to
              different destinations. Courier quotes are based on weight
              categories. Sign up to start enjoying our delivery service!
            </p>
            <Button
              path="/signup"
              classId={"home-sign-up"}
              text={"SIGN UP"}
              className={"fa fa-user-plus"}
            />
          </div>
          <div className="col-7">
            <img
              src="src/assets/home.png"
              alt="Happy client"
              className="home"
            />
          </div>
        </div>
      </div>
      <Footer />
    </div>
  );
};

export default HomePage;
