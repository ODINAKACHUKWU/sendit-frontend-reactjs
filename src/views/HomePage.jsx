import React from "react";
import { HomePageNavBar } from "../components/common/NavBar";
import Footer from "../components/common/Footer";
import Button from "../components/common/Button";
import homeImg from "../assets/home.png";

import "../styles/Homepage.css";

const HomePage = () => {
  return (
    <div className="row content">
      <HomePageNavBar />
      <div className="main content">
        <div className="home-container">
          <div className="col-5 left">
            <h1>Courier Services</h1>
            <p>
              SendIT is a courier service that helps users deliver parcels to
              different destinations. Courier quotes are based on weight
              categories. Sign up to start enjoying our delivery service!
            </p>
            <Button
              path="/signup"
              classId="home-sign-up"
              text="SIGN UP"
              className="fa fa-user-plus"
            />
          </div>
          <div className="col-7">
            <img
              src={homeImg}
              alt="Happy client"
              className="home"
            />
          </div>
        </div>
      </div>
      <Footer />
    </div>
  );
};

export default HomePage;
