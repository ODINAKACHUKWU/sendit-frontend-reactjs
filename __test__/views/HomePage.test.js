import React from "react";
import { shallow } from "enzyme";
import HomePage from "../../src/views/HomePage";

const component = shallow(<HomePage />);

describe("HomePage", () => {
  it("should render homepage", () => {
    expect(component).toMatchSnapshot();
  });
});
