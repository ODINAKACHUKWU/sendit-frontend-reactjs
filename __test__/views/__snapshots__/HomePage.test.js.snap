// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage should render homepage 1`] = `
<div
  className="row content"
>
  <HomePageNavBar />
  <div
    className="main content"
  >
    <div
      className="home-container"
    >
      <div
        className="col-5 left"
      >
        <h1>
          Courier Services
        </h1>
        <p>
          SendIT is a courier service that helps users deliver parcels to different destinations. Courier quotes are based on weight categories. Sign up to start enjoying our delivery service!
        </p>
        <Button
          classId="home-sign-up"
          className="fa fa-user-plus"
          path="/signup"
          text="SIGN UP"
        />
      </div>
      <div
        className="col-7"
      >
        <img
          alt="Happy client"
          className="home"
          src="src/assets/home.png"
        />
      </div>
    </div>
  </div>
  <Footer />
</div>
`;
